src/Adapter/ApcCache.php

```diff
@@ -17,6 +17,7 @@
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
 use Symfony\Component\Routing\RouterInterface;
+use function function_exists;
```

Comment:
what is this for?

Reply:
https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/ alternative is to use `\function_exists` this was looking cleaner to me, but if we are not doing this I can remove it.

Reply:
I think we are using the cs-fixer for that and it adds the `\`

Reply:
will fix it to `\` then.